tools/ctl/ctl.go

```diff
@@ -209,7 +211,97 @@ var getResults = &cobra.Command{
 			if err != nil {
 				log.Fatal(errors.Wrap(err, "marshalling benchmark"))
 			}
-			fmt.Println(string(b))
+			cmd.OutOrStdout().Write(b)
+			cmd.OutOrStderr().Write([]byte("\n"))
+		default:
+			log.Fatalf("Unknown --format type: %s", *format)
+		}
+	},
+}
+
+var missingAttestations = &cobra.Command{
```

Comment:
hmm this is a different PR, right? i think this should be separable from this change.